brave/src/test/java/brave/features/handler/RedactingFinishedSpanHandlerTest.java

```diff
@@ -59,40 +63,69 @@ static String maybeUpdateValue(String value) {
     }
   }
 
-  List<Span> spans = new ArrayList<>();
+  BlockingQueue<Span> spans = new LinkedBlockingQueue<>();
+  FinishedSpanHandler redacter = new FinishedSpanHandler() {
+    @Override public boolean handle(TraceContext context, MutableSpan span) {
+      span.forEachTag(ValueRedactor.INSTANCE);
+      span.forEachAnnotation(ValueRedactor.INSTANCE);
+      return true;
+    }
+
+    @Override public boolean supportsOrphans() {
```

Comment:
Just to make sure do we have a test that does gc and makes sure orphaned spans aren't processed by non-supporting handlers?

Reply:
we have a test that the Tracing object will not put unsupported handlers in the wrong place. https://github.com/openzipkin/brave/pull/972/files#diff-bce4e281981560a5111f2416f5f3413bR265